ephypype/commands/neuropycon.py

```diff
@@ -0,0 +1,319 @@
+"""Command line interface for ephypype package"""
+# TODO: autocompletion
+#       consider merging conn and ep2ts nodes
+#       create additional node for plugin option so
+#       the -n option can be invoked only in multiproc
```

Comment:
could you move this TODO list to an issue and instead add the author names + license here like we do in MNE: https://github.com/mne-tools/mne-python/blob/master/mne/epochs.py#L5-L11

Reply:
Ok, sure.

Reply:
Done